src/urbit/app/chess.hoon

```diff
@@ -879,6 +911,29 @@
                       ~
                   ==
               ==
+            %chess-resign
+              =/  result
+                ?:  =(our +.white.game.u.game-state)
+                  %'1-0'
+                %'0-1'
+              :-  :~  :*  %give
+                          %fact
+                          ~[/game/(scot %da u.game-id)/updates]
+                          %chess-update
+                          !>([%result u.game-id result])
+                      ==
+                      :*  %give  %kick  :~  /game/(scot %da u.game-id)/updates
+                                            /game/(scot %da u.game-id)/moves
+                                        ==
+                          ~
+                      ==
+                  ==
+              =/  updated-game  game.u.game-state
+              =.  result.updated-game  `result
+              %=  this
+                games    (~(del by games) u.game-id)
+                archive  (~(put by archive) u.game-id updated-game)
+              ==
```

Comment:
This new mark isn't necessary - you can just send the result as a `chess-game-result`. In the `chess-game-result` code, check if the result is a resignation by the other player. If so, accept it. If it's a draw, run the existing verification code. Otherwise, ignore it (nice try, cheater).